Gemfile.lock

```diff
@@ -437,20 +437,20 @@ GEM
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rack-timeout (0.6.0)
-    rails (6.1.4.1)
-      actioncable (= 6.1.4.1)
-      actionmailbox (= 6.1.4.1)
-      actionmailer (= 6.1.4.1)
-      actionpack (= 6.1.4.1)
-      actiontext (= 6.1.4.1)
-      actionview (= 6.1.4.1)
-      activejob (= 6.1.4.1)
-      activemodel (= 6.1.4.1)
-      activerecord (= 6.1.4.1)
-      activestorage (= 6.1.4.1)
-      activesupport (= 6.1.4.1)
+    rails (6.1.4.4)
```

Comment:
Interesting, Rails started to add a fourth version identifier. I reckon that it's technically seen as a pre-release like 7.0.1.alpha. That's probably why Dependabot isn't picking it up.

Reply:
Yeah it's a bit weird. Bundler said it was resolving `~> 6.1.4` to `6.1.4.1` even though the higher versions were available. 

I only noticed the new version was available because a Github security notice suggested upgrading `activesupport` to `6.1.4.4`. 

:man_shrugging: